api/v1alpha1/drpolicy_types.go

```diff
@@ -54,6 +54,22 @@ type DRPolicySpec struct {
 // DRPolicyStatus defines the observed state of DRPolicy
 type DRPolicyStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	//+optional
+	Async AsyncSpec `json:"async,omitempty"`
+	//+optional
+	Sync SyncSpec `json:"sync,omitempty"`
+}
+
+// for RDR
+type AsyncSpec struct {
```

Comment:
We should rename these `[Async|Sync]Spec` to a more relevant status field, than a spec field.

I would suggest we name these as follows:
- `[Async|Sync]Status`

Although the name will not appear in the YAML or JSON contents, it is clearer when using it in code.

Reply:
yes, I have changed it under DRPolicy. however VRG field is being used in many places and it was touching more files. thinking to taking that change in a separate PR.

Reply:
done